pkg/clusteragent/languagedetection/patcher_test.go

```diff
@@ -46,409 +47,16 @@ func newMockLanguagePatcher(ctx context.Context, mockClient dynamic.Interface, m
 		k8sClient: mockClient,
 		store:     mockStore,
 		logger:    mockLogger,
+		queue: workqueue.NewRateLimitingQueue(
```

Comment:
Is there a test that checks we're retrying ?

Reply:
I have just added one